docs/development/compatibility-policy.rst

```diff
@@ -551,6 +572,20 @@ Making calls to the deprecated code without raising these warnings can be done u
 
             self.assertEqual('some-value', user.homePath)
 
+        def test_getUserNonexistentDatabase(self):
+            """
+            The nesting of callDeprecated with assertRaises can result in ugly code.
```

Comment:
Per https://jml.io/test-docstrings/ this should say what is being tested, not how ugly we consider the aesthetics of the result